Readme.md

```diff
@@ -218,6 +216,30 @@ public class SecondSubCommand : ICommand
 }
 ```
 
+### Cancellation
+
+It is possible to gracefully cancel execution of a command and preform any necessary cleanup. By default an app gets forcefully killed when it receives an interrupt signal (Ctrl+C or Ctrl+Break). Make call to `console.RegisterCancellation()` to override the default behavior and get `CancellationToken` that represents the first interrupt signal. Second interrupt signal terminates an app immediately. 
```

Comment:
Some suggestions:

`Make call to` -> `You can call`

Reply:
Also, maybe we should mention that if the app receives interrupt signal before `RegisterCancellation` then it will terminate immediately

Reply:
Fixed.